EIPS/eip-7745.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 7745
+title: Gas Fee Sponsorship
+description: Enable one Ethereum address to sponsor gas fees for another by authorizing and co-signing transactions.
+author: Toni Dumančić (@toni-d-e-v)
+discussions-to: https://ethereum-magicians.org/t/eip-7745/7765
```

Comment:
Please create an actual topic in Eth Magicians with a link to this PR.

Reply:
I don't have an option to create anything on the website 😥

Reply:
There should be a "New Topic" button on the right hand side on Eth Magicians. 
![image](https://github.com/ethereum/EIPs/assets/28278242/9f207691-ad75-439e-9cc1-cee082186a98)

If this doesn't show, you may need to spend some time on the site first reading other posts.

Reply:
Yeah, could you just create it and send me the link.